ibis/expr/operations/relations.py

```diff
@@ -78,6 +78,67 @@ def resolve_name(self):
         return self.name
 
 
+@public
+class LocalTable(PhysicalTable):
+    schema = rlz.instance_of(sch.Schema)
+    name = rlz.instance_of(str)
+
+
+@public
+class LocalWrite(TableNode, sch.HasSchema):
```

Comment:
I'm not sure if this should be modeled as a `TableNode`.

The thing that determines whether a node should be a table node is whether the operation can **produce** rows.

From your perspective, does this or should this operation produce rows?

Reply:
Yes, it produces rows by passing-through. Not related to UDT; please skip.